ceed-fortran.c

```diff
@@ -64,41 +64,41 @@ void fCeedVectorCreate(int *ceed, int *length, int *vec, int *err) {
 
 #define fCeedVectorSetArray FORTRAN_NAME(ceedvectorsetarray,CEEDVECTORSETARRAY)
 void fCeedVectorSetArray(int *vec, int *memtype, int *copymode,
-    CeedScalar *array, int *err) {
+    double *array, int *err) {
   *err = CeedVectorSetArray(CeedVector_dict[*vec], *memtype, *copymode, array);
 }
 
 #define fCeedVectorGetArray FORTRAN_NAME(ceedvectorgetarray,CEEDVECTORGETARRAY)
 //TODO Need Fixing, double pointer
-void fCeedVectorGetArray(int *vec, int *memtype, CeedScalar *array, int *err) {
-  CeedScalar *b;
+void fCeedVectorGetArray(int *vec, int *memtype, double *array, int *err) {
```

Comment:
Why this change? We'll have to revisit every such interface whenever we enable changing the `CeedScalar` typedef.

Reply:
Derp, this was a mistake. I undid it.